chart_pod_template_file/tests/pod-template-file_test.yaml

```diff
@@ -0,0 +1,149 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+templates:
+  - pod-template-file.yaml
```

Comment:
Is it not possible to have this in the "normal" airflow `chart/` and have this as `files/pod-template-file.yaml`?

Reply:
@ashb I wish. The problem is that helm-unittest can ONLY test files in a "templates" folder. So if we keep it there, we can't unittest it.

Reply:
@ashb wait I think I figured something out

Reply:
Nice